pkg/test/bootguard_tests.go

```diff
@@ -26,40 +27,40 @@ var (
 		SpecificationDocumentID: IntelFITSpecificationDocumentID,
 	}
 	testbootguardacm = Test{
-		Name:                    "SACM meets sane BootGuard requirements",
-		Required:                true,
-		function:                BootGuardACM,
-		dependencies:            []*Test{&testbootguardfit},
```

Comment:
I like the alignemet before the change more :)

Reply:
this comes from the Go formatter, which my IDE ran automatically... :sweat_smile:

Reply:
I realized that this happened due to the comments. So, I just removed them. :)